src/components/table/index.njk

```diff
@@ -37,20 +35,160 @@
       }
     ],
     "rows": [
-    [
-      {
-        text: 'data'
-      },
-      {
-        text: 'object'
-      },
-      {
-        text: 'Yes'
-      },
-      {
-        text: 'Object containing array and object of data and options'
-      }
-    ]
+      [
+        {
+          text: 'data'
+        },
+        {
+          text: 'Object'
+        },
+        {
+          text: 'Yes'
+        },
+        {
+          text: 'Object containing all configuration options listed below'
+        }
+      ],
+      [
+        {
+          text: 'data.options'
+        },
+        {
+          text: 'Object'
+        },
+        {
+          text: 'No'
+        },
+        {
+          text: 'Options to set caption and its size as well as type of first cell im a row'
+        }
+      ],
+      [
+        {
+          text: 'data.options.caption'
+        },
+        {
+          text: 'String'
+        },
+        {
+          text: 'No'
+        },
+        {
+          text: 'Specify caption text if the table needs one'
+        }
+      ],
+      [
+        {
+          text: 'data.options.captionSize'
+        },
+        {
+          text: 'String'
+        },
+        {
+          text: 'No'
+        },
+        {
+          text: 'Specify caption size, relates to heading sizes'
+        }
+      ],
+      [
+        {
+          text: 'data.options.isFirstCellHeader'
+        },
+        {
+          text: 'Boolean'
+        },
+        {
+          text: 'No'
+        },
+        {
+          text: 'If true, the first cell in a row will be embolden'
```

Comment:
It's not about the styling – it's about the first cell in each row being marked up as a header for the rest of the row.